### PR TITLE
Renaming and refactoring of displaying videos

### PR DESCRIPTION
**Pull request by 3lvis**

### CODE REVIEW

File: Source/VideoView.swift [hunk 1]
```diff
@@ -6,13 +6,13 @@ import AVKit
     import Photos
 #endif
 
-protocol MovieContainerDelegate: class {
-    func movieContainerDidStartedPlayingMovie(_ movieContainer: MovieContainer)
-    func movieContainer(_ movieContainer: MovieContainer, didRequestToUpdateProgressBar duration: Double, currentTime: Double)
+protocol VideoViewDelegate: class {
```
Comment (elland) [inline Source/VideoView.swift:9]: Good rename. 👍

File: Source/VideoView.swift [hunk 2]
```diff
@@ -6,13 +6,13 @@ import AVKit
...
+    func videoViewDidStartedPlayingMovie(_ videoView: VideoView)
```
Comment (elland) [inline Source/VideoView.swift]: Grammar, please. DidStart, not DidStarted.

File: Source/VideoView.swift [hunk 3]
```diff
@@ -198,6 +196,6 @@ class MovieContainer: UIView {
     }
 
     func updateProgressBar(forDuration duration: Double, currentTime: Double) {
-        self.viewDelegate?.movieContainer(self, didRequestToUpdateProgressBar: duration, currentTime: currentTime)
+        self.viewDelegate?.videoView(self, didRequestToUpdateProgressBar: duration, currentTime: currentTime)
```
Comment (elland) [inline Source/VideoView.swift:199]: Much better name. Could be better? Is this the view that  displays the actual video, right? Maybe `videoPreview`?

**Review by elland — CHANGES_REQUESTED** [review]
Only issue I see from this code is the naming of a method.

**Comment by elland** [comment]
I see that the PR is to fix the iOS rotation thing, but all the changes look cosmetic to me (renames, clean-up), which line(s) actually fix the rotation problem?

File: Source/VideoView.swift (hunk 2 above)
Comment (3lvis) [reply Source/VideoView.swift]: Fixed!

File: Source/VideoView.swift (hunk 3 above)
Comment (3lvis) [reply Source/VideoView.swift:199]: Is not a preview, though, is the entire thing.

File: Source/ViewerController.swift [hunk 4]
```diff
@@ -428,11 +428,11 @@ extension ViewerController: ViewerItemControllerDelegate {
 }
 
 extension ViewerController: ViewerItemControllerDataSource {
-    func overlayIsHidden() -> Bool {
+    func viewerItemControllerIsOverlayHidden(_ viewerItemController: ViewerItemController) -> Bool {
```
Comment (elland) [inline Source/ViewerController.swift]: that reads worse. Try `isOverlayHidden` or `isViewerItemOverlayHidden`.

File: Source/ViewerItemController.swift [hunk 5]
```diff
@@ -293,7 +293,7 @@ extension ViewerItemController: UIScrollViewDelegate {
 }
 
 extension ViewerItemController: VideoViewDelegate {
-    func videoViewDidStartedPlayingMovie(_ videoView: VideoView) {
+    func videoViewDidStartPlayingMovie(_ videoView: VideoView) {
```
Comment (elland) [inline Source/ViewerItemController.swift]: Also if it's video everywhere, why is it still `movie` here? 👍

File: Source/VideoView.swift (hunk 3 above)
Comment (elland) [reply Source/VideoView.swift:199]: `videoView` is fine then. 👍

File: Source/ViewerController.swift (hunk 4 above)
Comment (3lvis) [reply Source/ViewerController.swift]: This needs better naming or maybe a whole refactoring. I'll start with one of your suggestions.

File: Source/ViewerItemController.swift (hunk 5 above)
Comment (3lvis) [reply Source/ViewerItemController.swift]: Fixed 👍

File: Source/VideoView.swift (hunk 3 above)
Comment (3lvis) [reply Source/VideoView.swift:199]: 👍

**Comment by 3lvis** [comment]
Updated this PR to only contain renames and clean up.

**Comment by elland** [comment]
👌 better

**Pull request merged**